src/content/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx

```diff
@@ -188,815 +188,67 @@ The following are proposed time ranges for EOL on older Node.js versions. The ac
-
-        <td>
-        </td>
-      </tr>
-
-      <tr>
-        <td>
-          [Hapi](https://www.npmjs.com/package/@hapi/hapi)
-        </td>
-
-        <td>
-        `@hapi/hapi`
-        </td>
-
-        <td>
-          20.0.0
-        </td>
-
-        <td>
-          9.0.0
-        </td>
-
-        <td>
-          With v9.0.0 of the agent we updated the minimum supported version of Hapi to be >= v20.0.0. All versions under v20.0.0 are deprecated by Hapi for security reasons, see their [support policy](https://hapi.dev/policies/support/).
-        </td>
-      </tr>
-
-      <tr>
-        <td>
-          [Koa](https://www.npmjs.com/package/koa)
-        </td>
-
-        <td>`koa`</td>
-
-        <td>
-          2.0.0
-        </td>
-
-        <td>3.2.0</td>
-
-        <td>
-            [External module](https://github.com/newrelic/node-newrelic-koa) loaded with the agent
-        </td>
-      </tr>
-
-      <tr>
-        <td>
-          [Fastify](https://www.npmjs.com/package/fastify)
-        </td>
-
-        <td>`fastify`</td>
-
-        <td>
-          2.0.0
-        </td>
-
-        <td>
-          8.5.0
-        </td>
-
-        <td>
-        </td>
-      </tr>
-
-      <tr>
-        <td>
-          [Nest.js](https://nestjs.com/)
-        </td>
-
-        <td>`@nestjs/core`</td>
-
-        <td>
-          8.0.0
-        </td>
-
-        <td>
-          10.1.0
-        </td>
-
-        <td>
-          If using the `nest start` command to start the application, modify its startup binary to load the New Relic agent: `nest start --exec 'node -r newrelic'`
-        </td>
-      </tr>
-
-      <tr>
-        <td>
-          [Next.js](https://www.npmjs.com/package/next)
-        </td>
-
-        <td>`next`</td>
-
-        <td>
-          12.0.9 - 13.3.0
-        </td>
-
-        <td>
-          10.1.0
-        </td>
-
-        <td>
-          v12.2.0 or higher required for middleware instrumentation
-        </td>
-      </tr>
-
-    </tbody>
-  </table>
-
-If you are using a supported framework with default routers, the Node.js agent can read these frameworks' route names as is. However, if you want more specific names than are provided by your framework, you may want to use one or more of the tools New Relic provides with the [Node.js transaction naming API](/docs/nodejs/nodejs-transaction-naming-api).
-
-  <Callout title="EOL NOTICE">
-    We've discontinued support for several capabilities in November 2021. This includes the Oracle Driver Package and Hapi versions prior to Hapi 19.2 for our Node.js agent. For more details, including how you can prepare for this transition, see our [Support Forum post](https://discuss.newrelic.com/t/important-upcoming-changes-to-capabilities-and-support-across-node-agents-suggested-pagerduty-responders-incident-workflows-and-kubernetes-instrumentation/164481?u=dholloran).
-  </Callout>
-</Collapser>
-</CollapserGroup>
-
-## Connect the agent to other New Relic features [#digital-intelligence-platform]
```

Comment:
this section will be needed. Some may be cleaned up but other links are unrelated to libraries we support